src/i18n/de-de/index.js

```diff
@@ -0,0 +1,358 @@
+            fetch_stlos_apy_error: 'Abruf der sTLOS APY fehlgeschlagen: { message }',
+            fetch_max_deposits_error: 'Abruf der maximalen Abhebungen aus ESCROW Smar Contract fehlgeschlagen: { message }',
+            convert_tlos_to_stlos_error: 'TLOS konnte nicht in STLOS umgewandelt werden: { message }',
+            convert_stlos_to_tlos_error: 'sTLOS konnte nicht in TLOS umgewandelt werden:S { message }',
+            unstake_stlos_error: 'sTLOS konnte nicht abgehoben werden: { message }',
+            withdraw_failed: 'TLOS Position konnte nicht ausgezahlt werden: { message }',
+        },
+        explore_transactions: 'Transaktionen prüfen',
+        recent_transactions: 'Neueste Transaktionen',
+        telos_evm_explorer: 'Telos EVM Explorer',
+        rpc_endpoints: 'RPC Endpunkte',
+        monitor: 'Monitor',
+        oops: 'Oh. Hier existiert nichts...',
+        go_home: 'Home',
+        select_sol_file: 'Bitte .sol Smart Contract Datei für Hochladen auswählen',
+        select_json_file: 'Bitte standard JSON Objekt-Datei für Hochladen auswählen',
+        paste_contract_contents: 'Sie müssen eine Datei zum Hochladen auswählen oder den Reiter wechseln, um Smart Contract Inhalte direkt einzufügen.',
+        contract_address: 'Smart Contract Adresse',
+        enter_contract_address: 'Bitte Smart Contract Adresse eingeben \'0x0123...\'',
+        invalid_address_format: 'Ungültiges Adress-Format',
+        compiler_version: 'Compiler Version',
+        select_compiler_version: 'Compiler Version auswählen',
+        eg_contracts: 'd.h.:, \'contracts/\'',
+        contract_file_directory_path: 'Smart Contract Datei(en) Verzeichnis Pfad (leer lassen, falls nicht vorhanden)',
+        invalid_path_format: 'Der Pfad muss mit einem Schrägstrich enden /',
+        upload_file: 'Datei hochladen',
+        text_input: 'Texteingabe',
+        runs_value_for_optimization: 'Führt Wert zur Optimierung aus',
+        constructor_arguments: 'Constructor Arguments',
+        comma_seperated_values: 'Kommagetrennte Werte z.B.:, Bob,123,0x12345...',
+        no_trailing_commas: 'Keine nachgestellten Kommas',
+        paste_contract_code_here: 'Smart Contract Code kopieren und hier einfügen...',
+        enter_contract_text: 'Smart Contract Code eingeben oder einfügen',
+        verify_contract: 'Smart Contract verifizieren',
+        reset: 'Zurücksetzen',
+        gas_used: 'Verbrauchtes Gas',
+        transactions: 'Transaktionen',
+        account: 'Konto',
+        contract: 'Smart Contract',
+        erc20_transfers: 'ERC20 Transfers',
+        erc721_transfers: 'ERC721 Transfers',
+        erc1155_transfers: 'ERC1155 Transfers',
+        tokens: 'Tokens',
+        created_at_trx: 'Erstellt in Transaktion',
+        by_address: 'Durch Konto/Adresse',
+        number_used_once: 'Einmalig verwendete Nummer (nonce)',
+        native_account: 'Konto: Telos Native',
+        balance: 'Saldo',
```

Comment:
🤯

Reply:
Anything I need to do here? ;-)

Reply:
not at all! just impressed with the unexpected same term